webui/src/components/Files.jsx

```diff
@@ -11,13 +11,18 @@ require('dropzone/dist/min/dropzone.min.css');
 
 
 const log = bows('Files');
+const BuildProgress = ({isInProgress, statusMessage}) => {
+    if (isInProgress == false) {
```

Comment:
should be `if (!isInProgress) }`, which would match even when isInProgress is undefined